docs/user/faq/troubleshooting.md

```diff
@@ -2,106 +2,118 @@
+</ul>
+
+If all of the above does not work, visit our <a href="https://discord.com/invite/dappnode">Discord</a> and ask for help in the #support channel.
+</div>
+  </div>
+</details>
+
+<details>
+  <summary>Why can't I connect to my DAppNode via VPN?</summary>
+  <div>
+    <div>
+    If you have set up OpenVPN and you cant access your DAppNode, remember that you might need to set up port forwarding on your router. Sometimes the feature called 'UPnP' enables these ports on your router automatically but this sometimes does not work or is not available on your router.
+    <br /><br />
+
+To do this, you need to access your router configuration and "port-forward" the following ports: TCP Port 8092 and UDP Port 1194. 
+
+In case you have set up Wireguard, you might have to create two profiles:
+
+<ul>
+    <li>A Local profile: This profile is used when you are connecting from within the same network that the DAppNode resides in.</li>
+    <li> A Remote profile: This profile is used when you are connecting from a different network than the one where your DAppNode resides in.</li>
+</ul>
+  When you create the VPN profiles, there is a link that generates the profile for either local usage or remote usage.
+  <br /><br />
+You can also check out our guides on how to configure VPN access <a href="/user/guides/access/vpn">here</a>.
+</div>
+  </div>
+</details>
+
+<details>
+  <summary>Can I recover a lost admin password?</summary>
+  <div>
+    <div>
+    You can not outright recover your admin password but you can regain access to your DAppNode if you wrote down the recovery token provided during the inital setup. 
+    <br /><br />
+    More about the initial setup can be found <a href="/user/quick-start/first-steps">here</a>
+    <br /><br />
+</div>
+  </div>
+</details>
+
+<details>
+  <summary>How do I recover a lost admin password using my recovery token?</summary>
+  <div>
+    <div>
+    Visit the login page of your DAppNode:
+    <br /><br />
+    <p align="center">
     <img src="../../../img/login.png"/>
-</p>
-
-We should select the option "Forgot password?". It will appear a field where you can reset your password using the recovery token. In this example, we would use the token <code>4LMB9w3l50Yljwr6bIgQ</code>. You have to use your recovery token.
+    </p>
 
+You will then see the option 'Forgot password?' underneath the 'Login' button. Once you click that, a new input field will appear where you can input your recovery token. We will use <code>4LMB9w3l50Yljwr6bIgQ</code> in this example.
 <p align="center">
     <img src="../../../img/reset_password_with_recovery_token.png"/>
 </p>
-
-After confirming the recovery token, you can define an admin user in DAppNode and his password in the same way you did the first time you connect.
-
+If you have entered the correct recovery token, you'll then be prompted to enter a new admin password.
+<br /><br />
 <p align="center">
     <img src="../../../img/sign_in_image.png"/>
```

Comment:
```suggestion
    <img src="../../../static/img/sign_in_image.png"/>
```

Reply:
Running docusaurus with `yarn start` locally displays the images without an issue with the current path (no static in path).